app/assets/stylesheets/views/_browse.scss

```diff
@@ -329,3 +329,84 @@ $transition-speed: .4s;
     }
   }
 }
+
+// New background colour for the topic pages:
+$topic-page-header-background-colour: #263135;
+
+// This isn't proper BEM - but the convention is to have the view name as the
+// block part of the class name. Because we need to keep `browse` so this can
+// be run as an A/B test, we need to use a different class name for each view.
+.topic-page__breadcrumb-wrapper,
+.topic-page__header-wrapper {
+  background-color: $topic-page-header-background-colour;
+}
+
+.topic-page__breadcrumb-wrapper {
+  // Setting the overflow to auto is required to prevent the breadcrumb
+  // component's margin from collapsing.
+  overflow: auto;
+
+  // We want to show a constrained width blue bar _within_ the grey header.
+  // But:
+  //  * The full width coronavirus banner currently shows the blue bar - but
+  // this is only visible when JavaScript is available.
+  //  * A full width layout template doesn't include the blue bar.
+  //
+  // So we need to add a blue bar manually to this template - but only when
+  // JavaScript is unavailable.
+  //
+  // When the coronavirus banner is turned off, there will be some work to turn
+  // this on for both JS and non-JS visitors in a way that doesn't break any
+  // future banner or the emergency banner.
```

Comment:
The coronavirus banner has been turned off - are these styles still needed?

Reply:
For reference, @injms is now going to look into this.